docs/setup.rst

```diff
@@ -46,7 +46,7 @@ used to launch user's Python script with the debugging scaffolding in place to
 communicate with Mu, acting as the debug client). As a result, it's essential
 to run the following to ensure this command is available in your virtualenv::
 
-  python setup.py develop
+  pip install --editable .
```

Comment:
This is the pip equivalent of `python setup.py develop`.
There might be some subtle differences, but since we are pretty much using pip for every step here I though it might be better to use pip here as well.